mauricio/proyecto1/js/main.js

```diff
@@ -60,37 +59,55 @@
    * @return void
    */
   function closeModal($event) {
-    console.log($event.target.parentElement.parentElement.style.display = 'none');
+    document.querySelector('.modal').remove();
   }
 
   /**
-   * @description pinta en un modal la ubicación del evento en Google Maps
-   * @param {string} idEvent - id que servirá para obtener las coordenadas del evento
-   * @param data.results.venue: array - array de todas las direcciones
-   * @param event.street - dirección del evento para el DOM
-   * @param google.maps.Map - API google maps
-   * @param google.maps.Marker - API google maps
+   * @description muestra u oculta el spinner con css dependiendo de si el state es show o no
+   * @param {string} si es show, aplica 'block' a display, en otro caso, 'none'
    * @return void
    */
+  function toggleSpinner(state) {
+    const spinner = document.querySelector('.spinner');
+    let css = spinner.style;
+    if (state === 'show') {
```

Comment:
Podría ser con un if inline... `spinner.style.display = state === 'show' ? 'block' : 'none'` ;-)

Reply:
Al principio lo tenía en una línea pero `jshint` me decía _Expected an assignment or function call and instead saw an expression_ y acabo de caer que es porque hacía falta el `return`. Corregido!